cards/p/seznam-poslancev/data.json

```diff
@@ -8734,73 +8734,6 @@
         }
       }
     },
-    {
-      "person": {
-        "name": "Sun\u010dana Glavak",
```

Comment:
This lady made it impossible to load the card - @LepkoQQ am I missing something?

Reply:
Yes, she doesn't seem to have any data (see all the nulls). So it probably tries to load something there.

Reply:
Yeah, I get that's why it fails, what I'm wondering is - am I missing some configuration that would allow me to open this card or is it possible that the stuff in data.json is bad?

Reply:
maybe the problem is you have slovenian sources set up at one place and croatian sources someplace else. check config. it looks to me like your data.json is croatian - where did you get it from?